api.wsgi

```diff
@@ -24,85 +28,96 @@ ap.add_argument('--api_uri', help='api uri, with https:// prefix')
 ap.add_argument('--site_id', help='site ID for Scitran Central [local]', default='local')
 ap.add_argument('--site_name', help='site name', nargs='*', default=['Local'])  # hack for uwsgi --pyargv
 ap.add_argument('--oauth2_id_endpoint', help='OAuth2 provider ID endpoint', default='https://www.googleapis.com/plus/v1/people/me/openIdConnect')
-ap.add_argument('--demo', help='enable automatic user creation', action='store_true', default=False)
 ap.add_argument('--insecure', help='allow user info as urlencoded param', action='store_true', default=False)
 ap.add_argument('--central_uri', help='scitran central api', default='https://sdmc.scitran.io/api')
 ap.add_argument('--log_level', help='log level [info]', default='info')
+ap.add_argument('--drone_secret', help='shared drone secret')
+
+if __name__ == '__main__':
+    import paste.httpserver
+    logging.getLogger('paste.httpserver').setLevel(logging.WARNING) # silence paste logging
+
+    ap.add_argument('--host', default='127.0.0.1', help='IP address to bind to [127.0.0.1]')
+    ap.add_argument('--port', default='8080', help='TCP port to listen on [8080]')
```

Comment:
Are these conditionally-added arguments only for launching the app without a wsgi server?